janitor/functions.py

```diff
@@ -3142,9 +3133,7 @@ def currency_column_to_numeric(
 @pf.register_dataframe_method
 @deprecated_alias(search_cols="search_column_names")
 def select_columns(
-    df: pd.DataFrame,
-    search_column_names: Union[str, callable, Pattern, slice, list],
-    invert: bool = False,
+    df: pd.DataFrame, *args, invert: bool = False,
```

Comment:
@samukweku this is smaht (Bostonian accent)! I like the API change. It's both backwards-compatible _and_ makes the API much more flexible.